dlt/destinations/job_client_impl.py

```diff
@@ -431,15 +428,22 @@ def _commit_schema_update(self, schema: Schema, schema_str: str) -> None:
 
 
 class SqlJobClientWithStaging(SqlJobClientBase, WithStagingDataset):
+
+    in_staging_mode: bool = False
```

Comment:
IMO we'll not need it

Reply:
the problem is, that in the place where I need this information (building the tables), the table info is being taken from the schema and not passed into the function. I could pass down this information via update_stored_schema, but this will not be much nicer than this hook i'm afraid..